db/schema.rb

```diff
@@ -192,6 +192,16 @@
     t.index ["run_id"], name: "index_run_histories_on_run_id"
   end
 
+  create_table "run_likes", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.bigint "run_id", null: false
+    t.bigint "user_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["run_id", "user_id"], name: "index_run_likes_on_run_id_and_user_id", unique: true
```

Comment:
With the indices, doesn't having the combined index make the 2 separate ones obsolete?